### ISSUE TITLE
`make check` fails on i386 (2.1.1)

### ISSUE THREAD

**Issue by jengelh** [libsemigroups/libsemigroups#323]
make check fails on i386/i586.

```
[  421s] [0m-------------------------------------------------------------------------------
[  421s] [0mMultiStringView 000: catch all
[  421s] [0m-------------------------------------------------------------------------------
[  421s] [0;37mtests/test-string-view.cpp:37
[  421s] ...............................................................................
[  421s] 
[  421s] [0m[0;37mtests/test-string-view.cpp:41: [0m[1;31mFAILED:
[  421s] [0m[0;36m  REQUIRE( sizeof(detail::MultiStringView) == 32 )
[  421s] [0mwith expansion:
[  421s] [1;33m  20 == 32
[  421s] [0m
[  421s] [1;37mMultiStringView 000: catch all . . .[0m                                           [1;37m
[  421s] [0m[1;31m=[0m[1;33m[0m[0;32m==============================================================================[0m
[  421s] test cases:       711[0;37m | [0m[0;32m      710 passed[0m[0;37m | [0m[1;31m1 failed[0m
[  421s] assertions: 1,096,837[0;37m | [0m[0;32m1,096,836 passed[0m[0;37m | [0m[1;31m1 failed[0m
```

**Comment by james-d-mitchell** [comment]
Ugh, same issue as the last one, I think. Thanks for the report, I'll resolve this today.

@jengelh is this the only test that fails?

**Comment by jengelh** [comment]
It would seem so. `test cases: 711, 710 passed, 1 failed`

**Comment by james-d-mitchell** [comment]
Ok, thanks a lot for the reports!

**Comment by james-d-mitchell** [comment]
Hopefully I resolved this in v2.1.2. Thanks again @jengelh
